panel/chat/__init__.py

```diff
@@ -52,9 +52,10 @@ def __getattr__(name):
     """
     Lazily import langchain module when accessed.
     """
+    import importlib as _importlib
+
     if name == "langchain":
-        from . import langchain
-        return langchain
+        return _importlib.import_module("panel.chat.langchain")
```

Comment:
```suggestion
        import importlib
        return importlib.import_module("panel.chat.langchain")
```

And delete the import outside of the if statement.